ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/console/components/packages/groupSelect/groupSelect.html

```diff
@@ -2,7 +2,7 @@
 <div class="coral-Form-fieldwrapper">
     <label class="coral-Form-fieldlabel">Group</label>
 
-    <coral-select id="indexTypeSelect" placeholder="Select Group" name="group" class="coral-Form-field">
+    <coral-select id="indexTypeSelect" placeholder="Select Group" name="group" class="coral-Form-field" value="Select Group">
```

Comment:
Will it mean, that now, if user don't do anything to the list, the package will be saved to a group named "Select group"?)